src/VideoStreaming/VideoStreaming.cc

```diff
@@ -28,6 +28,8 @@
 #endif
 
 #include "VideoStreaming.h"
+#include "VideoSurface.h"
+#include "VideoManager.h"
```

Comment:
`VideoReceiver.h`?

Reply:
VideoManager is the current class that initializes the VideoReceiver. one of the things I plan to do is to merge two classes. let's see.